### PR TITLE
fix(CurrentDeploymentPanel): support otp2 graph downloads

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Co-authored-by: philip-cline <philip-cline@users.noreply.github.com>

### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

OTP2 uses `g`raph rather than `G`raph

### CODE REVIEW

**Review by AdrianaCeric — APPROVED** [review]
Nice. May the otp2 graph downloads be ever in your favour

File: lib/manager/components/deployment/CurrentDeploymentPanel.js [hunk 1]
```diff
@@ -229,7 +229,8 @@ class DeployJobSummary extends Component<{
   }
 
   _onClickDownloadGraph = () => {
-    this.props.downloadBuildArtifact(this.props.deployment, 'Graph.obj', this._getJobId())
+    const g = this.props.deployment.tripPlannerVersion === 'OTP_2' ? 'g' : 'G'
+    this.props.downloadBuildArtifact(this.props.deployment, `${g}raph.obj`, this._getJobId())
```
Comment (daniel-heppner-ibigroup) [inline lib/manager/components/deployment/CurrentDeploymentPanel.js:233]: Creating a whole variable for the single letter feels a little bit much. Couldn't we just have a `filename` variable that is either Graph.obj or graph.obj?

**Review by daniel-heppner-ibigroup — CHANGES_REQUESTED** [review]

File: lib/manager/components/deployment/CurrentDeploymentPanel.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/manager/components/deployment/CurrentDeploymentPanel.js:233]: Yes but that would be more

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Convincing argument

**Pull request merged**